src/conn/conn_api.c

```diff
@@ -1224,9 +1223,12 @@ __conn_close(WT_CONNECTION *wt_conn, const char *config)
         F_SET(conn, WT_CONN_LEAK_MEMORY);
 
     /* Time since the shutdown has started. */
-    __wt_timer_evaluate(session, &timer, &time_diff);
+    __wt_timer_evaluate(session, &timer, &conn->shutdown_timeline.shutdown);
     __wt_verbose(session, WT_VERB_RECOVERY_PROGRESS,
-      "shutdown has successfully finished and ran for %" PRIu64 " seconds", time_diff);
+      "shutdown was completed successfully and took %" PRIu64 " ms, including %" PRIu64
```

Comment:
Nitpick: we were using "seconds" but moved to "ms", so we should lose some of the spaces in this log line - i.e. "10 seconds" -> "10000ms". (I can see you're using "milliseconds" further down, that can keep the space)

Reply:
Thanks, fixed.